cfgov/paying_for_college/documents.py

```diff
@@ -0,0 +1,60 @@
+from django_elasticsearch_dsl import Document, fields
+from django_elasticsearch_dsl.registries import registry
+from elasticsearch_dsl import analyzer, token_filter, tokenizer
+
+from paying_for_college.models import School
+
+
+label_autocomplete = analyzer(
```

Comment:
We shouldn't duplicate our analyzers and filters, maybe define a generic `elasticsearch_helpers.py` file in the search app.